.gitignore

```diff
@@ -140,6 +140,9 @@ cython_debug/
 # Pycharm project settings
 .idea
 
+# VSCode project settings
+/.vscode
```

Comment:
Let's remove this, instead you can add your editor files into a [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).